modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -108,6 +108,20 @@ CV__DNN_INLINE_NS_BEGIN
         DNN_TARGET_NPU,
     };
 
+    /**
+     * @brief Enum of data layout for model inference.
+     * @see Image2BlobParams
+     */
+    enum DataLayout
+    {
+        DNN_LAYOUT_UNKNOWN = 0,
+        DNN_LAYOUT_ND = 1,
+        DNN_LAYOUT_NCHW = 2,      //!< OpenCV and ONNX data layout.
+        DNN_LAYOUT_NHWC = 3,      //!< Tensorflow-like data layout.
+        DNN_LAYOUT_NDHWC = 4,      //!< Tensorflow-like data layout.
```

Comment:
Difference between `DNN_LAYOUT_NDHWC` and `DNN_LAYOUT_NHWC` is not clear.

Reply:
The `NDHWC` is for tensorflow conv3d, and the `NHWC` is tensorflow Conv2D.
I will add details documentation for this part code.